### ISSUE TITLE
Too much unnecessary

### ISSUE THREAD

**Issue by MuhammedHasan** [Haylaz01/Assignment1#1]
There is too much unnecessary code in project.
Are you get payment for each line. :D

**Comment by Haylaz01** [comment]
Yeah I know there is too much unnecessary code but it is just for homework and we assumed to know this style of coding :D, payment case maybe.
